Extension/package.json

```diff
@@ -286,6 +286,15 @@
           "description": "The value to use in a configuration if \"macFrameworkPath\" is not specified, or the values to insert if \"${default}\" is present in \"macFrameworkPath\".",
           "scope": "resource"
         },
+        "C_Cpp.default.windowsSdkVersion": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "default": null,
+          "description": "Version of the Windows SDK include path to use on Windows, e.g. '10.0.17134.0'.",
+          "scope": "resource"
```

Comment:
Should the pattern go in here too?

Reply:
Hmm...it doesn't seem to be required...I could try adding it.